theia-docker/Dockerfile

```diff
@@ -30,5 +30,8 @@ COPY --from=0 --chown=theia:theia /home/theia /home/theia
 EXPOSE 3000
 ENV SHELL /bin/bash
 ENV USE_LOCAL_GIT true
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
```

Comment:
I got a permission issue at first. Got it working with a little tweak, so that the `theia` user has execute permission as well:

`RUN chmod 555 /tini`